docs/lib/auth/fragments/js/social.md

```diff
@@ -493,18 +494,26 @@ const urlOpener = async (url, redirectUrl) => {
 
 const oauth = {
     // Domain name
-    domain : 'your-domain-prefix.auth.us-east-1.amazoncognito.com', 
+    domain : 'your-domain-prefix.auth.us-east-1.amazoncognito.com',
 
     // Authorized scopes
-    scope : ['phone', 'email', 'profile', 'openid','aws.cognito.signin.user.admin'], 
+    scope : ['phone', 'email', 'profile', 'openid','aws.cognito.signin.user.admin'],
 
     // Callback URL
-    redirectSignIn : 'http://www.example.com/signin/', // or 'exp://127.0.0.1:19000/--/', 'myapp://main/'
+    redirectSignIn : 'http://www.example.com/signin/',
+    // For React Native via Expo in development
+    // redirectSignIn : 'exp://127.0.0.1:19000/--/',
+    // For React Native in production
+    // redirectSignIn : 'myapp://main/',
```

Comment:
This would also be the URL for apps created with React Native CLI in development, right?